--- cssutils-0.9a6/setup.py.wiget	2006-10-29 18:04:46.000000000 +0100
+++ cssutils-0.9a6/setup.py	2006-11-02 18:06:45.000000000 +0100
@@ -7,8 +7,8 @@
 __docformat__ = 'restructuredtext'
 __version__ = '0.9a6'
 
-import ez_setup
-ez_setup.use_setuptools()
+#import ez_setup
+#ez_setup.use_setuptools()
 
 from setuptools import setup
 
@@ -43,4 +43,4 @@
     keywords='CSS, Cascading Style Sheets, CSSParser, DOM Level 2 Stylesheets, DOM Level 2 CSS',
     download_url='http://cthedot.de/cssutils/',
     url='http://cthedot.de/cssutils/'
-    )
\ Brak znaku nowej linii na końcu pliku
+    )
